setup.cfg

```diff
@@ -88,6 +88,8 @@ markers = serial
 ; Filter google rcp related deprecation warnings.
 ; See https://github.com/googleapis/python-api-common-protos/issues/23
 filterwarnings =
+    error
+    ignore:Exception ignored in
```

Comment:
these two are so generic :) it a plan to remove them before merge or ?

Reply:
I think you have misunderstood how this works. The first line means that all warnings that do not match the ignore lines will be an error so it should be generic. It would indeed be nice to make the next one less generic but I am not sure it is possible since the next char in the error string is a : and there seems to be no way to escape that in this ignore dsl

Reply:
indeed, sorry for misunderstanding the first line. for the second line, the docs say:
> message is a string containing a regular expression that the start of the warning message must match. The expression is compiled to always be case-insensitive

so perhaps escpaing the colon in a regexp way, or making a regexp that ignores the "beginning of the string" default behavior (say `*.`) and reflects the actual error message would work?

Reply:
Removed this from the pr